NvidiaGPU/resources.json

```diff
@@ -85,13 +93,21 @@
               "Type" : "GRID",
               "Version" : [
                 {
-                  "Num" : "425.31",
-                  "DirLink" : "http://download.microsoft.com/download/6/D/7/6D73C628-B5FB-4243-9520-DAEF363223CB/425.31_grid_win8_win7_server2012R2_server2008R2_64bit_international.exe",
+                  "Num" : "431.79",
+                  "DirLink" : "https://download.microsoft.com/download/8/6/e/86ef2daa-b31e-43ad-90f2-bd795384b71e/431.79_grid_win7_win8_server2008R2_server2012R2_64bit_international.exe",
                   "FwLink" : "https://go.microsoft.com/fwlink/?linkid=874184"
                 },
+                {
+                  "Num" : "431.02",
+                  "FwLink" : "http://download.microsoft.com/download/B/D/6/BD6D1C34-E41F-4654-B5AB-F4DA9C08AA64/431.02_grid_win7_win8_server2008R2_server2012R2_64bit_international.exe"
+                },
+                {
+                  "Num" : "425.31",
+                  "FwLink" : "http://download.microsoft.com/download/6/D/7/6D73C628-B5FB-4243-9520-DAEF363223CB/425.31_grid_win8_win7_server2012R2_server2008R2_64bit_international.exe"
+                },
                 {
                   "Num" : "412.16",
-                  "DirLink" : "http://download.microsoft.com/download/0/9/8/098B5E8A-0563-4B25-9CDB-173B03B52C60/412.16_grid_win8_win7_server2012R2_server2008R2_64bit_international.exe"
+                  "FwLink" : "http://download.microsoft.com/download/0/9/8/098B5E8A-0563-4B25-9CDB-173B03B52C60/412.16_grid_win8_win7_server2012R2_server2008R2_64bit_international.exe"
```

Comment:
same, this is dirlinks

Reply:
FwLinks are being used according to Aman